how-to-guides/how-it-works/notebooks/Neptune_API_Tour.ipynb

```diff
@@ -427,7 +463,13 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {},
+   "metadata": {
+    "colab": {
+     "base_uri": "https://localhost:8080/",
+     "height": 1000
+    },
+    "outputId": "d3726875-4a6a-4c38-d904-bdfd415c97cc"
+   },
```

Comment:
@SiddhantSadangi please include that metadata in the pre-commit -> nbstripout.

I see it in few places.

Reply:
I mean `metadata.colab` and `metadata.outputId`

Reply:
These are the metadata which are being stripped currently: cell.metadata.colab cell.metadata.id cell.metadata.outputId cell.metadata.pycharm metadata.kernelspec metadata.vscode